components/image-placeholder/ImageWithPlaceholder.tsx

```diff
@@ -27,6 +27,7 @@ const ImageWithPlaceholder = ({ src, alt }: IProps) => {
         className={`${loaded ? 'opacity-100' : 'opacity-0'} h-0 pb-[calc((8/17)*100%)]`}
       >
         <Image
+          suppressHydrationWarning
```

Comment:
why do we get a hydration warning here? if we can't fix it, we should add a comment to explain it

Reply:
here is only because the loading of the random pictures on the profile. as the randomness will be done via the date.getMinutes. it can be that the server renders something different than the client. In a real world case the user would habe a ProfileImageBIG or something similar. we can avoid it by adding just a common pic like it was before.